origami.json

```
@@ -1,6 +1,6 @@
 {
-	"description": "Utility for formatting and updating dates",
-	"keywords": "datetime time timestamp",
+	"description": "JavaScript utilities for formatting and updating dates in FT style. Useful for formatting dates to express relative time.",
```

Thought it might sound a bit like it only does relative times without the also?
```suggestion
	"description": "JavaScript utilities for formatting and updating dates in the FT style. Also useful for formatting dates relative to the current time.",
```